### ISSUE TITLE
Make the core of Terminal Emulator a library project

### ISSUE THREAD

**Issue by corbinlc** [jackpal/Android-Terminal-Emulator#381]
I know this project is closed to feature requests, but I am wondering if it would be worthwhile for many to make the core of the terminal emulator a library project.  This would allow other developers to more easily integrate this functionality into their apps.

**Comment by steven676** [comment]
I'm confused -- are you asking for something other than https://github.com/jackpal/Android-Terminal-Emulator/tree/master/emulatorview ?

**Comment by corbinlc** [comment]
No.  In my use, I am using code that is not part of the emulatorview as well.  I am effectively using the entirety of android terminal emulator and then marking that a library project which is in turn dependent on the emulatorview library project.  I thought that would be useful for others.
